cf_cli_java_plugin_test.go

```diff
@@ -177,13 +175,31 @@ var _ = Describe("CfJavaPlugin", func() {
 						output, err := subject.DoRun(commandExecutor, uuidGenerator, pluginUtil, []string{"java", "heap-dump", "my_app"})
 						return output, err
 					})
-
+					// heapDumpFile := pluginUtil.Fspath + "/my_app-heapdump-" + pluginUtil.UUID + ".hprof"
 					Expect(output).To(BeEmpty())
 					Expect(err).To(BeNil())
-					Expect(cliOutput).To(Equal(""))
+					// Empty output is not produced in any case now.
+					Expect(cliOutput).To(Equal("Successfully created heap dump in application container at: " + pluginUtil.Fspath + "/" + pluginUtil.OutputFileName + "|Heap dump will not be copied as parameter `local-dir` was not set|Heap dump file deleted in app container|"))
 
 					Expect(commandExecutor.ExecuteCallCount()).To(Equal(1))
-					Expect(commandExecutor.ExecuteArgsForCall(0)).To(Equal([]string{"ssh", "my_app", "--command", JavaDetectionCommand + "; if [ -f /tmp/heapdump-abcd-123456.hprof ]; then echo >&2 'Heap dump /tmp/heapdump-abcd-123456.hprof already exists'; exit 1; fi; JMAP_COMMAND=`find -executable -name jmap | head -1 | tr -d [:space:]`; if [ -n \"${JMAP_COMMAND}\" ]; then true; OUTPUT=$( ${JMAP_COMMAND} -dump:format=b,file=/tmp/heapdump-abcd-123456.hprof $(pidof java) ) || STATUS_CODE=$?; if [ ! -s /tmp/heapdump-abcd-123456.hprof ]; then echo >&2 ${OUTPUT}; exit 1; fi; if [ ${STATUS_CODE:-0} -gt 0 ]; then echo >&2 ${OUTPUT}; exit ${STATUS_CODE}; fi; cat /tmp/heapdump-abcd-123456.hprof; exit 0; fi; JVMMON_COMMAND=`find -executable -name jvmmon | head -1 | tr -d [:space:]`; if [ -n \"${JVMMON_COMMAND}\" ]; then true; OUTPUT=$( ${JVMMON_COMMAND} -pid $(pidof java) -c \"dump heap\" ) || STATUS_CODE=$?; sleep 5; HEAP_DUMP_NAME=`find -name 'java_pid*.hprof' -printf '%T@ %p\\0' | sort -zk 1nr | sed -z 's/^[^ ]* //' | tr '\\0' '\\n' | head -n 1`; SIZE=-1; OLD_SIZE=$(stat -c '%s' \"${HEAP_DUMP_NAME}\"); while [ \"${SIZE}\" != \"${OLD_SIZE}\" ]; do sleep 3; SIZE=$(stat -c '%s' \"${HEAP_DUMP_NAME}\"); done; if [ ! -s \"${HEAP_DUMP_NAME}\" ]; then echo >&2 ${OUTPUT}; exit 1; fi; if [ ${STATUS_CODE:-0} -gt 0 ]; then echo >&2 ${OUTPUT}; exit ${STATUS_CODE}; fi; cat ${HEAP_DUMP_NAME}; fi; rm -f /tmp/heapdump-abcd-123456.hprof; if [ -n \"${HEAP_DUMP_NAME}\" ]; then rm -f ${HEAP_DUMP_NAME} ${HEAP_DUMP_NAME%.*}.addons; fi"}))
+
+					/*
```

Comment:
This was changed in a recent commit: https://github.com/SAP/cf-cli-java-plugin/commit/036a5eacddab82bf519139d598b28875ddf73fa0#diff-5aa2577361069a3cc3969893474e77c83b3af621509137e5cba9e8b57fee82b8L192
Due to the tests not having been executed, this was not caught. The executed command below looks like what should be expected here.

Reply:
suggested change has been incorporated in code.